launcher/modplatform/flame/FlamePackExportTask.h

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-3.0-only
+/*
+ *  Prism Launcher - Minecraft Launcher
+ *  Copyright (c) 2023 Trial97 <alexandru.tripon97@gmail.com>
```

Comment:
this looks like it was pasted from ModrinthPackExportTask

Reply:
Yeah it was pasted from there(only changed the Copyright line. What should I change?

Reply:
I thought you probably need to keep the copyright line. i don't personally mind though!

Reply:
If you want I can add you there :smile_cat:

Reply:
Point is.. I think you may need to do this

Reply:
sure done